### PR TITLE
Fix UI on Bulksettings Page

### PR DESCRIPTION
**Pull request by albertliangcode**
Repaired minor errors in how text was being displayed on page.

After this fix:
- Text wraps around to correct indent
- Max Attempts setting defaults to "-"
- Weight setting defaults to "1"
- Subsection due dates and types default to "N/A"
- Problem names default to "Unnamed Problem"

### CODE REVIEW

**Comment by albertliangcode** [comment]
Please review, @stvstnfrd!

**Comment by stvstnfrd** [comment]
@albertliangcode Yeah, I think you were overly zealous in squashing down this request.

Per the story, this PR is making 3+ semi-distinct changes, though upon squashing them together, we lose much of that context. For example, what do the CSS changes have to do with changing the default weight to "1"?

If if it's a bit confusing to _me_, _now_, after just having lent you a hand during development, imagine how much more opaque this will look to someone stumbling upon it months from now.

I'd like to split each of the UI changes out into their own commits; I can give some pointers on this.

File: cms/templates/bulksettings.html [hunk 1]
```diff
@@ -59,23 +59,36 @@ <h4 class = "section-header-title" title="Collapse/Expand this utility">
...
                           <strong><a href="${subsection_settings['url']}">${subsection_settings['name']}</a></strong> -
                           % for setting_type in setting_type_list_map['subsection_setting_types']:
-                              <span title=${setting_type} class="subsection-settings"><strong>${subsection_setting_map[setting_type]}:</strong> ${subsection_settings[setting_type]}</span>
+                              <%
+                              subsection_setting = subsection_settings[setting_type] or "N/A"
+                              %>
+                              <span title=${setting_type} class="subsection-settings"><strong>${subsection_setting_map[setting_type]}:</strong> ${subsection_setting}</span>
                           % endfor
-                        </td>
+                                </td>
```
Comment (stvstnfrd) [inline cms/templates/bulksettings.html]: I don't think you meant to check this in..

File: cms/templates/bulksettings.html [hunk 2]
```diff
@@ -59,23 +59,36 @@ <h4 class = "section-header-title" title="Collapse/Expand this utility">
                                 <td class = "subsection-header-cell" colspan = "7">
                           <strong><a href="${subsection_settings['url']}">${subsection_settings['name']}</a></strong> -
                           % for setting_type in setting_type_list_map['subsection_setting_types']:
-                              <span title=${setting_type} class="subsection-settings"><strong>${subsection_setting_map[setting_type]}:</strong> ${subsection_settings[setting_type]}</span>
+                              <%
+                              subsection_setting = subsection_settings[setting_type] or "N/A"
```
Comment (stvstnfrd) [inline cms/templates/bulksettings.html]: Please use single quotes here and throughout for consistency.

Comment (stvstnfrd) [inline cms/templates/bulksettings.html]: Clarification: Use single quotes for non-interpolated Python strings.

File: cms/templates/bulksettings.html [hunk 3]
```diff
@@ -59,23 +59,36 @@ <h4 class = "section-header-title" title="Collapse/Expand this utility">
...
                                     <td width = "8%">${unit_settings['ispublic']}</td> <!--hard coded right now-->
                                 </tr>
 
                                 % for problem_settings in unit_settings['children']:
                                     <tr class="problem-header">
-                                        <td width="52%"><strong><a href="${problem_settings['url']}">${problem_settings['name']}</a></strong></td>
+                                        <%
+                                        problem_name = problem_settings['name'] or "Unnamed Problem"
+                                        %>
+                                        <td width="52%" class="problem-title"><strong><a href="${problem_settings['url']}">${problem_name}</a></strong></td>
```
Comment (stvstnfrd) [inline cms/templates/bulksettings.html]: The variable is being called `problem_name` but then the class is called `problem_title`.

Is one more apt than the other?

Should we decide on one (title or name) for consistency?

**Comment by stvstnfrd** [comment]
@albertliangcode I'd like you to demo this for @nthaniel to confirm that this meets the expectations of his story; please reach out to coordinate this with him.

**Comment by albertliangcode** [comment]
Okay, @stvstnfrd!  I'll make the changes to the code first - once it's pushed and ready for another round of review, I'll try to demo for @nthaniel.

**Comment by stvstnfrd** [comment]
@albertliangcode Well, those things don't necessarily need to happen in that order. In fact, I'd recommend doing the "product walkthrough" ASAP and _not_ wait until it's passed code review because if @nthaniel has suggesstions and/or you didn't quite implement something the way it was designed, we need to find that out sooner. Else, you risk needing to rewrite the code and then the time we spent reviewing code could be wasted if that code needs changed anyway.

**Comment by albertliangcode** [comment]
@stvstnfrd - Got it!

File: cms/templates/bulksettings.html (hunk 1 above)
Comment (albertliangcode) [inline cms/templates/bulksettings.html]: Sorry - the </td> didn't look aligned with the <td> for subsection-header-cell.  Should I move the </td> back to its old spot?

File: cms/templates/bulksettings.html (hunk 3 above)
Comment (albertliangcode) [inline cms/templates/bulksettings.html]: Okay!  I'll rename `problem_name` to be `problem_title`.

File: cms/static/sass/views/_bulksettings.scss [hunk 4]
```diff
@@ -71,6 +47,27 @@
         tr td {
             padding: 5px;
         }
+
+        .subsection-header {
```
Comment (albertliangcode) [inline cms/static/sass/views/_bulksettings.scss]: The CSS diff for text wrap looks like a massive block of changes, instead of the small series of swaps that actually happened.  Should I split this commit into 2+ commits to show this more clearly, or is it still understandable?

File: cms/templates/bulksettings.html (hunk 1 above)
Comment (stvstnfrd) [inline cms/templates/bulksettings.html]: No, for 2 reasons:
- the alignment for that first `<TD>` is actually wrong (indented too much), so we don't want to mimic it.
- that change is unrelated to this feature.

Comment (stvstnfrd) [inline cms/templates/bulksettings.html]: Clarification: Yes you should revert the change.

Comment (albertliangcode) [inline cms/templates/bulksettings.html]: OK!

**Comment by stvstnfrd** [comment]
+1 from me.

@caesar2164 Mind looking over the CSS/HTML?

**Comment by albertliangcode** [comment]
Robot, test this please.

**Comment by albertliangcode** [comment]
retest this please

**Comment by caesar2164** [comment]
As the HTML and the CSS is basically what I showed you last week, I'm :+1: :smile:

**Comment by albertliangcode** [comment]
Okay!  Merging now!

**Comment by albertliangcode** [comment]
Merged and branches cleaned.  Thanks a lot!
